Source/Core/Core/PowerPC/JitCommon/JitBackpatch.cpp

```diff
@@ -71,6 +71,22 @@ bool Jitx86Base::BackPatch(u32 emAddress, SContext* ctx)
 		return false;
 	}
 
+	auto it2 = pcAtLoc.find(codePtr);
+	u32 pc;
+	if (it2 == pcAtLoc.end())
+	{
+		if (info.isMemoryWrite || jit->js.memcheck)
+		{
+			PanicAlert("BackPatch: no pc entry for address %p", codePtr);
```

Comment:
Wait, what?  We never write the PC to SPR_DAR.

The following code saves the PC before we jump to the exception handler:
```
				// If a memory exception occurs, the exception handler will read
				// from PC.  Update PC with the latest value in case that happens.
				MOV(32, PPCSTATE(pc), Imm32(ops[i].address));
```